src/index.css

```diff
@@ -1,4 +1,294 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@700&family=Roboto+Slab&display=swap');
```

Comment:
- [ ] Excellent job for using the correct `font-family` 💪 However, the correct `font-weight` is still not applying to the elements  despite that you are using the given `font-weight` values⚠ This is because only only `700` is being imported. As an example, see the difference in the below images 👇
You can use `@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@200;300;400;500;600;700;800&family=Roboto+Slab:wght@200;300;400;500;600;700;800&display=swap');` or a different one. In any case, please make sure that `font-weight` is applied correctly for getting a replica of the Zeplin's design 🤓⛏

Your design | Given design 
---| ---|
![image](https://github.com/lorenzoworx/bookstore-react/assets/106119105/3b714382-b9fb-4a29-bb97-d09f12e2bb79) | ![image](https://github.com/lorenzoworx/bookstore-react/assets/106119105/00f7812b-da09-4e89-a9e5-59d69667b822)

_Note: The above comparison is just an example, all other elements are behaving the same way, please fix this so that the entire design is according to the Zeplin's given design_